x-pack/plugins/security_solution/cypress/tasks/integrations.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ADD_INTEGRATION_BTN,
+  INTEGRATION_ADDED_POP_UP,
+  QUEU_URL,
+  SAVE_AND_CONTINUE_BTN,
+} from '../screens/integrations';
+
+import { visit } from './login';
+
+export const installAwsCloudFrontWithPolicy = () => {
+  visit('app/integrations/detail/aws-1.17.0/overview?integration=cloudfront');
```

Comment:
I was curious if this would be stable going forward once a new version of this package is released, and looks like it should be fine as the button to install is still there as expected. Only mentioning as the UI can change slightly for older versions of the same package (mostly just the `Settings` page it looks like):

v1.1.0:
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/2946766/177431939-e5187c4e-e745-4e1c-a391-caf1aa6207e3.png">

v1.4.0 (latest):
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/2946766/177432015-0c0505d8-c8dd-4cd0-831e-1b98a45e1c06.png">

Reply:
I think it should be stable, if you check the URL, we are pointing to a specific version (in the case of the test 1.17.0), so I guess we are not going to face big UI changes.